src/app/components/LiteSiteCta/index.styles.tsx

```diff
@@ -0,0 +1,62 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../utilities/pixelsToRem';
+
+export default {
+  outerContainer: ({ palette }: Theme) =>
+    css({
+      backgroundColor: `${palette.WHITE}`,
+      borderBottom: `${pixelsToRem(1)}rem solid ${palette.GREY_3}`,
+    }),
+  container: ({ spacings, mq }: Theme) =>
+    css({
+      margin: `0 ${spacings.FULL}rem`,
+      padding: `${spacings.TRIPLE}rem 0 ${spacings.FULL}rem 0`,
+      maxWidth: '63.4rem',
+      position: 'relative',
+      [mq.GROUP_2_MIN_WIDTH]: {
+        margin: `0 ${spacings.DOUBLE}rem`,
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        margin: `0 auto`,
+      },
+    }),
+  chevron: ({ palette, spacings }: Theme) =>
```

Comment:
If I increase the browser font size the chevron doesn't increase in size, it should do

Reply:
Good spot, having added height & width css to the chevron.

(Note an alternative is nesting it in the text element. However we can't easily do this since we don't want a bottom border to appear underneath the chevron. So it's easier to add height & width to it.)